### ISSUE TITLE
ViewPlugin (and its module TabComplete) check is not working

### ISSUE THREAD

**Issue by dustdustinn** [CraftG/Horizon#24]
The check "ViewPlugin" and its module "TabComplete" seems not to work.
without any bypass permissions on the default rank  the player is still able to bypass the check.
Check seems not be loaded or work with the new MC mechanics

INFOMATION:
Server: Spigot
Server version:  1.13.2-R0.1-SNAPSHOT
ISSUE TYPE:  bug

**Comment by CraftG** [comment]
Yes, due to new tab mechanics in 1.13. This will be fixed in the next version.
